package.json

```diff
@@ -47,7 +47,7 @@
     "ts-node": "^4.1.0",
     "tslint": "^5.9.1",
     "tslint-microsoft-contrib": "^5.0.1",
-    "typescript": "2.6.2",
+    "typescript": "^2.7.1",
```

Comment:
no ^

Reply:
Then that should be everywhere, we've used caret everywhere